### ISSUE TITLE
[Request] Make BibleTime respect $XDG_CONFIG_HOME

### ISSUE THREAD

**Issue by endlesslake** [bibletime/bibletime#275]
Currently, BibleTime just puts `.bibletime` and `.sword` directly in `~`. I like to keep my home directory tidy, so it would be great if BibleTime would use `$XDG_CONFIG_HOME` instead of just `~`.

**Comment by jaakristioja** [comment]
I totally agree. However, please note that we do not control the location of `~/.sword`. Although there are some ugly non-portable tricks we could use to force libsword use other locations (e.g. via modifying the `HOME` environment variable or using `ptrace`), none are worth to integrate into BibleTime. Please contact the Crosswire libsword developers for `~/.sword`.

**Comment by jaakristioja** [comment]
Actually, using only `$XDG_CONFIG_HOME` would probably not be enough. I think it makes more sense to adhere to the whole [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/latest/), which also includes environment variables like `$XDG_DATA_HOME`, `$XDG_DATA_DIRS`, `$XDG_CONFIG_DIRS`, `$XDG_CACHE_HOME`, `$XDG_RUNTIME_DIR`.

**Comment by jaakristioja** [comment]
This would however require some kind of automatic migration mechanisms as well. And it must also be noted, that this might does not apply to Windows and possibly some other platforms.
